apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -5,6 +5,7 @@
 import com.codahale.metrics.Slf4jReporter;
 import com.codahale.metrics.jvm.GarbageCollectorMetricSet;
 import com.codahale.metrics.jvm.MemoryUsageGaugeSet;
+import com.newrelic.agent.deps.com.google.common.base.Strings;
```

Comment:
We should be able to pick this up directly from com.google rather than going through new relic.

Reply:
fixed